seungjong/services/postService.js

```diff
@@ -0,0 +1,43 @@
+// services/postService.js
+
+const postDao = require('../models/postDao');
+
+const addPost = async (title, content, imgUrl, userId) => {
+    const addPost = await postDao.addPost(
+        title,
+        content,
+        imgUrl,
+        userId
+    );
+
+    return addPost;
+}
+
+const posts = async () => {
+    
+    const posts = await postDao.posts();
+    return posts;
```

Comment:
```suggestion
    return postDao.posts();
```
불필요한 변수 선언 및 비동기처리(await)없이 곧바로 return해도 깔끔한 코드가 되겠네요!

이하 동일한 포맷의 다른 코드들도 마찬가지입니다~